api/external/envoy/api/v2/discovery.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package solo.io.envoy.api.v2;
```

Comment:
This rename could result in downtime for ext-auth/rate-limit so it's being reverted